src/idl_gen_js_ts.cpp

```diff
@@ -127,9 +127,7 @@ class JsTsGenerator : public BaseGenerator {
       const auto &file = *it;
       const auto basename =
           flatbuffers::StripPath(flatbuffers::StripExtension(file));
-      if (basename != file_name_) {
-        code += GenPrefixedImport(file, basename);
-      }
+      if (basename != file_name_) { code += GenPrefixedImport(file, basename); }
```

Comment:
Please don't put condition body in the same line as the condition itself, since it's hard to debug such code.

Reply:
fixed.